src/test/java/uk/gov/hmcts/ccd/endpoint/std/GlobalSearchEndpointIT.java

```diff
@@ -0,0 +1,188 @@
+package uk.gov.hmcts.ccd.endpoint.std;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+import org.springframework.web.context.WebApplicationContext;
+import uk.gov.hmcts.ccd.WireMockBaseTest;
+import uk.gov.hmcts.ccd.domain.model.search.global.GlobalSearchRequestPayload;
+import uk.gov.hmcts.ccd.domain.model.search.global.GlobalSearchSortByCategory;
+import uk.gov.hmcts.ccd.domain.model.search.global.GlobalSearchSortDirection;
+import uk.gov.hmcts.ccd.domain.model.search.global.Party;
+import uk.gov.hmcts.ccd.domain.model.search.global.SearchCriteria;
+import uk.gov.hmcts.ccd.domain.model.search.global.SortCriteria;
+import uk.gov.hmcts.ccd.domain.model.std.validator.ValidationError;
+
+import javax.inject.Inject;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+import static uk.gov.hmcts.ccd.endpoint.std.GlobalSearchEndpoint.GLOBAL_SEARCH_PATH;
+
+public class GlobalSearchEndpointIT extends WireMockBaseTest {
+
+    @Inject
+    private WebApplicationContext wac;
+    private MockMvc mockMvc;
+
+    public List<String> listOfValidFields;
+    public List<String> listOfInvalidFields;
+    public List<SortCriteria> listOfValidSortCriteria;
+    public List<SortCriteria> listOfInvalidSortCriteria;
+    public List<String> listOfValidCaseReferences;
+
+    @Before
+    public void setUp() throws IOException {
+        mockMvc = MockMvcBuilders.webAppContextSetup(wac).build();
+
+        listOfValidFields = new ArrayList<>();
+        listOfValidFields.add("ValidEntry");
+        listOfValidFields.add("ValidEntryTwo");
```

Comment:
```suggestion
        listOfValidFields = List.of("ValidEntry", "ValidEntryTwo");
```
Minor: Possible to simplify things like this in Java 9+ to a one-liner